histoqc/LightDarkModule.py

```diff
@@ -59,10 +59,25 @@ def getIntensityThresholdPercent(s, params):
     upper_thresh = float(params.get("upper_threshold", "inf"))
 
     # Prepare parameter names due to issues #213 and #219
+    
     # set lower standard deviation
-    lower_std = float(args.get("lower_var") or args.get("lower_std") or "-inf")
+    lower_var = params.get("lower_var")
+    lower_std = float(lower_var or params.get("lower_std") or "-inf")
+    if lower_var:
+        logging.warning(f"{s['filename']} - LightDarkModule.getIntensityThresholdPercent:{name} "
+                        f"lower_var will be deprecated in future versions in favor of lower_std")
+        s["warnings"].append(f"LightDarkModule.getIntensityThresholdPercent:{name} "
+                        f"lower_var will be deprecated in future versions in favor of lower_std")
+
     # set upper standard deviation
-    upper_std = float(args.get("upper_var") or args.get("upper_std") or "inf")
+    upper_var = params.get("upper_var")
+    if upper_var:
+        logging.warning(f"{s['filename']} - LightDarkModule.getIntensityThresholdPercent:{name} "
+                        f"upper_var will be deprecated in future versions in favor of upper_std")
+        s["warnings"].append(f"LightDarkModule.getIntensityThresholdPercent:{name} "
+                        f"upper_var will be deprecated in future versions in favor of upper_std")
+    
+    upper_std = float(upper_var or params.get("upper_std") or "inf")
```

Comment:
sorry sorry please don't hate me...but i loved your previous 1-liner code better : ) 

I didn't realize that it was possible to set the statement in a single line like:

```
 upper_std = float(args.get("upper_var") or args.get("upper_std") or "inf")
```

which is clearly more elegant to me than having this deprecation notice (which i know i asked for...my fault)

Can we revert this to what you had before and remove the deprecation notice?  :-\

Reply:
reverted it. ;)